### PR TITLE
Jirasearch (Independent Publisher)

### PR DESCRIPTION
**Pull request by paulculmsee**
---
When submitting a connector, please check the following conditions for your PR to ensure a smooth certification process.

- [x ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@diegobernardes Please resolve the branch conflicts to get the PR approved and merged

File: independent-publisher-connectors/JiraSearch/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+                                "type": "string",
+                                "description": "Project Key"
+                              },
+                              "name": {
+                                "type": "string",
+                                "description": "name"
+                              },
+                              "projectTypeKey": {
+                                "type": "string",
+                                "description": "projectTypeKey"
```
[the remark this answers is not in the document]
Comment (paulculmsee) [reply independent-publisher-connectors/JiraSearch/apiDefinition.swagger.json]: done... some of the API documentation is a bit lacking on the Jira side so hope this is better

**Comment by sriyen-msft** [comment]
@paulculmsee Please resolve the branch conflicts to move forward in the certification and deployment process.

**Comment by sriyen-msft** [comment]
@paulculmsee I am closing this PR as there is no update from you since more than 15 days. You can reopen this PR whenever you are ready for updating it. Thanks

**Pull request closed without merging**